providers/heyreach.go

```diff
@@ -0,0 +1,41 @@
+package providers
+
+const HeyReach Provider = "heyreach"
+
+func init() {
+	SetInfo(HeyReach, ProviderInfo{
+		DisplayName: "heyreach",
```

Comment:
Even though their logo is all lower case, I think the display name should probably be "HeyReach", since that's how it appears in text throughout their website, examples:
![Screenshot 2025-01-02 at 4 25 04 PM](https://githu
![Screenshot 2025-01-02 at 4 26 10 PM](https://github.com/user-attachments/assets/c5c87b69-d218-4941-9396-1703435e9293)
b.com/user-attachments/assets/3629f9ee-02a2-437f-969e-5b23dcc555f3)